src/store/mutators.ts

```diff
@@ -2,13 +2,14 @@ const mutate = require('immutability-helper');
 import { FlowNode, UINode, AnyAction, FlowDefinition, Dimensions, StickyNote } from '../flowTypes';
 import { v4 as generateUUID } from 'uuid';
 import { RenderNode, RenderNodeMap } from './flowContext';
-import { dump, snapToGrid } from '../utils';
+import { dump, snapToGrid, set, merge, unset, splice } from '../utils';
 import { getUniqueDestinations, getNode, getExitIndex, getActionIndex } from './helpers';
 import { LocalizationUpdates } from '.';
+import { push } from '../utils/index';
 
 export const uniquifyNode = (newNode: FlowNode): FlowNode => {
     // Give our node a unique uuid
-    return mutate(newNode, { $merge: { uuid: generateUUID() } });
+    return mutate(newNode, merge({ uuid: generateUUID() }));
```

Comment:
functions named `set`, `merge`, etc feel really overloaded, but willing to give it a go.

Reply:
Ha. Overloaded is somewhat overloaded itself ;) Generally it is used to refer to commonly named things that can confuse meaning, so for example, would also say "Action" is pretty overloaded in this codebase :)

Reply:
ahhh, gotcha thanks. perhaps we should change these to `setImmutably`, `mergeImmutably`, etc?